app/scripts/modules/ecs/pipeline/stages/scaleDownCluster/ecsScaleDownClusterStage.js

```diff
@@ -0,0 +1,68 @@
+'use strict';
+
+const angular = require('angular');
+
+module.exports = angular.module('spinnaker.ecs.pipeline.stage.scaleDownClusterStage', [
+])
+  .config(function(pipelineConfigProvider) {
+    pipelineConfigProvider.registerStage({
+      provides: 'scaleDownCluster',
+      cloudProvider: 'ecs',
+      templateUrl: require('./scaleDownClusterStage.html'),
+      executionConfigSections: ['scaleDownClusterConfig', 'taskStatus'],
```

Comment:
can remove `executionConfigSections` and the empty `*executionDetails.html` template.

Reply:
I removed the `executionConfigSections` and the executionDetails.html